### PR TITLE
Update babel version

### PR DESCRIPTION
**Pull request by tacigar**
Update babel version to babel 7.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
any motivation for this, except keeping current?

File: zipkin-lens/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  [
+    '@babel/env',
+    {
+      targets: {
+        edge: '17',
+        firefox: '60',
+        chrome: '67',
+        safari: '11.1',
+      },
+      useBuiltIns: 'usage',
```
Comment (tacigar) [inline zipkin-lens/babel.config.js:11]: `useBuiltIns` option configures how @babel/preset-env handles polyfills.

Please see:
* https://babeljs.io/docs/en/babel-preset-env#usebuiltins
* https://babeljs.io/docs/en/babel-preset-env#usebuiltins-usage (about `usage` option)

File: zipkin-lens/babel.config.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      corejs: 3,
```
Comment (tacigar) [inline zipkin-lens/babel.config.js:12]: If `useBuiltIns` options is specified, `corejs` options should be specified.
`3` is corejs version which we use.

Please see:
* https://babeljs.io/docs/en/babel-preset-env#corejs

File: zipkin-lens/babel.config.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+const presets = [
+  [
+    '@babel/env',
```
Comment (tacigar) [inline zipkin-lens/babel.config.js:3]: We can use shorthand name (whole name is `@babel/preset-env`).

Please see:
* https://babeljs.io/docs/en/v7-migration#package-renames

File: zipkin-lens/babel.config.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        edge: '17',
+        firefox: '60',
+        chrome: '67',
+        safari: '11.1',
+      },
+      useBuiltIns: 'usage',
+      corejs: 3,
+    },
+  ],
+  '@babel/react',
```
Comment (tacigar) [inline zipkin-lens/babel.config.js:15]: This is shorthand name too.
Whole name is `@babel/preset-react`.

File: zipkin-lens/package.json [hunk 5]
```diff
@@ -88,6 +88,7 @@
     "react-virtualized": "^9.21.1",
     "redux": "^4.0.0",
     "redux-thunk": "^2.3.0",
+    "regenerator-runtime": "^0.13.2",
```
Comment (tacigar) [inline zipkin-lens/package.json:93]: We need to install core-js & regenerator-runtime for using polyfill.

Please see: 
* https://github.com/zloirock/core-js/blob/master/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#usebuiltins-usage-with-corejs-3

File: zipkin-lens/package.json [hunk 6]
```diff
@@ -68,8 +68,8 @@
     "@material-ui/core": "^4.1.1",
     "@material-ui/pickers": "^3.1.1",
     "@material-ui/styles": "^4.1.1",
-    "babel-polyfill": "^6.26.0",
```
Comment (tacigar) [inline zipkin-lens/package.json:71]: `@babel/polyfill` (`babel-polyfill`) is deprecated.
Please see the top of this page (https://babeljs.io/docs/en/babel-polyfill)

File: zipkin-lens/src/index.js [hunk 7]
```diff
@@ -18,8 +18,6 @@ import App from './components/App';
 
 import '../scss/main.scss';
 
-require('babel-polyfill');
```
Comment (tacigar) [inline zipkin-lens/src/index.js:21]: We don't need import polyfill modules anymore.
Babel will do it automatically.

**Comment by tacigar** [comment]
Keeping current is main motivation.
And avoiding to import unused polyfill modules and reducing the amount of code are my motivation too.

**Review by codefromthecrypt — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
messages like this from travis 


```

[ERROR]     /home/travis/build/openzipkin/zipkin/zipkin-lens/src/components/MiniTimeline/MiniTimelineGraph.test.js: Unexpected token (23:6)
```

**Comment by tacigar** [comment]
I changed the babel config file from `.baberc` to `babel.config.js`.
But Jest did not read the new config file due to cache...
This is the cause of the above error.

I tried to test with `--ci` option.

**Comment by codefromthecrypt** [comment]
also you can delete cache in travis ui

**Comment by tacigar** [comment]
Ah, I see.
But test seems success now.

```
[ERROR] PASS src/zipkin/span-row.test.js
[ERROR] PASS src/zipkin/span-cleaner.test.js
[ERROR] PASS src/zipkin/trace.test.js
[ERROR] PASS src/zipkin/clock-skew.test.js
[ERROR] PASS src/zipkin/span-node.test.js
[ERROR] PASS src/components/DiscoverPage/api.test.js
[ERROR] PASS src/reducers/global-search.test.js
[ERROR] PASS src/components/App/SidebarMenuItem.test.js
[ERROR] PASS src/reducers/traces.test.js
[ERROR] PASS src/reducers/dependencies.test.js
[ERROR] PASS src/reducers/trace.test.js
[ERROR] PASS src/actions/traces-action.test.js
[ERROR] PASS src/actions/global-search-action.test.js
[ERROR] PASS src/actions/trace-action.test.js
[ERROR] PASS src/components/GlobalSearch/util.test.js
[ERROR] PASS src/actions/dependencies-action.test.js
[ERROR] PASS src/components/Browser/BrowserResults.test.js
[ERROR] PASS src/reducers/remote-services.test.js
[ERROR] PASS src/components/MiniTimeline/MiniTimeline.test.js
[ERROR] PASS src/components/App/Sidebar.test.js
[ERROR] PASS src/components/MiniTimeline/MiniTimelineSlider.test.js
[ERROR] PASS src/actions/remote-services-action.test.js
[ERROR] PASS src/reducers/spans.test.js
[ERROR] PASS src/components/App/App.test.js
[ERROR] PASS src/components/Browser/sorting.test.js
[ERROR] PASS src/components/MiniTimeline/MiniTimelineLabel.test.js
[ERROR] PASS src/util/trace.test.js
[ERROR] PASS src/actions/spans-action.test.js
[ERROR] PASS src/reducers/autocomplete-values.test.js
[ERROR] PASS src/reducers/autocomplete-keys.test.js
[ERROR] PASS src/reducers/services.test.js
[ERROR] PASS src/actions/autocomplete-values-action.test.js
[ERROR] PASS src/components/Browser/TraceSummary/TraceSummary.test.js
[ERROR] PASS src/actions/autocomplete-keys-action.test.js
[ERROR] PASS src/actions/services-action.test.js
[ERROR] PASS src/components/MiniTimeline/util.test.js
[ERROR] PASS src/components/Browser/Browser.test.js
[ERROR] PASS src/components/DiscoverPage/TraceIdSearchInput.test.js
[ERROR] PASS src/components/Browser/TraceSummary/TraceSummaryUpperBar.test.js
[ERROR] PASS src/actions/trace-viewer.test.js
[ERROR] PASS src/components/App/Layout.test.js
[ERROR] PASS src/components/Browser/TraceSummary/TraceSummaryBar.test.js
[ERROR] PASS src/components/Browser/TraceSummary/TraceSummaryButtons.test.js
[ERROR] PASS src/components/Browser/BrowserHeader.test.js
[ERROR] PASS src/components/MiniTimeline/MiniTimelineTimeMarkers.test.js
[ERROR] PASS src/components/MiniTimeline/MiniTimelineGraph.test.js
[ERROR] 
[ERROR] Test Suites: 46 passed, 46 total
[ERROR] Tests:       264 passed, 264 total
[ERROR] Snapshots:   0 total
[ERROR] Time:        17.383s
[ERROR] Ran all test suites.
```

The reason why log is ERROR is the spec of `frontend-maven-plugin`.

https://github.com/eirslett/frontend-maven-plugin/issues/584

File: zipkin-lens/package.json [hunk 8]
```diff
@@ -26,13 +26,14 @@
   "author": "https://gitter.im/openzipkin/zipkin",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@babel/cli": "^7.5.0",
+    "@babel/core": "^7.5.4",
+    "@babel/preset-env": "^7.5.4",
+    "@babel/preset-react": "^7.0.0",
     "@fortawesome/fontawesome-free": "^5.4.1",
-    "babel-core": "^6.26.3",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (tacigar) [inline zipkin-lens/package.json:35]: This package is necessary.

↓ Jest 23.6's document
https://github.com/facebook/jest/blob/770799779fc853ab6285727e8938efd06e694d5f/README.md#using-babel

**Comment by tacigar** [comment]
PTAL 🙇

**Comment by codefromthecrypt** [comment]
thank you!

**Pull request merged**